chain-core/src/init/address.rs

```diff
@@ -252,6 +252,13 @@ impl fmt::Display for RedeemAddress {
     }
 }
 
+#[cfg(feature = "hex")]
+impl fmt::Debug for RedeemAddress {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "(0x{}", hex::encode(self.0))
+    }
+}
+
```

Comment:
How about remove the `#[cfg(feature = "hex")]`?

Reply:
it won't compile, because the tx-validation enclave doesn't use chain-core with default features (so there's no hex crate)

Reply:
```error[E0277]: `init::address::RedeemAddress` doesn't implement `core::fmt::Debug`
651	  --> /tmp/drone-ZUPW93uRybLyAqBa/drone/src/chain-core/src/state/account.rs:45:17
652	   |
653	45 |     BasicRedeem(RedeemAddress),
654	   |                 ^^^^^^^^^^^^^ `init::address::RedeemAddress` cannot be formatted using `{:?}`
655	
656	   |
657	   = help: the trait `core::fmt::Debug` is not implemented for `init::address::RedeemAddress`
658	   = note: add `#[derive(Debug)]` or manually implement `core::fmt::Debug`
659	   = note: required because of the requirements on the impl of `core::fmt::Debug` for `&init::address::RedeemAddress`
660	   = note: required for the cast to the object type `dyn core::fmt::Debug`
661	
662	error: aborting due to previous error
663	
664	For more information about this error, try `rustc --explain E0277`.
665	error: Could not compile `chain-core`.```